gremlin-go/driver/connection_test.go

```diff
@@ -164,6 +164,9 @@ func TestConnection(t *testing.T) {
 	testHost := getEnvOrDefaultString("GREMLIN_SERVER_HOSTNAME", "localhost")
 	testPort := getEnvOrDefaultInt("GREMLIN_SERVER_PORT", 8182)
 	runIntegration := getEnvOrDefaultBool("RUN_INTEGRATION_TESTS", true)
+	// Set this to true to when TinkerGraphs are configured on the test server
+	// TODO: remove after setting up docker integration testing that automatically sets up TinkerGraphs on test server
+	runIntegrationWithAlias := false
```

Comment:
Should be obtained from environment with default like others

Reply:
Added